vips/vip-256/bscmainnet.ts

```diff
@@ -0,0 +1,343 @@
+  },
+  {
+    symbol: "Cake",
+    amount: BigNumber.from(parseUnits("3397.15865", 18)).add(parseUnits("891.64848", 18)).toString(),
+    address: "0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+  },
+  {
+    symbol: "UNI",
+    amount: parseUnits("1252.25385", 18).toString(),
+    address: "0xBf5140A22578168FD562DCcF235E5D43A02ce9B1",
+  },
+  {
+    symbol: "FDUSD",
+    amount: BigNumber.from(parseUnits("5261.65425", 18)).add(parseUnits("1031.64844", 18)).toString(),
+    address: "0xc5f0f7b66764F6ec8C8Dff7BA683102295E16409",
+  },
+  {
+    symbol: "DOT",
+    amount: BigNumber.from(parseUnits("618.32785", 18)).add(parseUnits("529.97468", 18)).toString(),
+    address: "0x7083609fCE4d1d8Dc0C979AAb8c869Ea2C873402",
+  },
+  {
+    symbol: "ADA",
+    amount: parseUnits("6877.765", 18).toString(),
+    address: "0x3EE2200Efb3400fAbB9AacF31297cBdD1d435D47",
+  },
+  {
+    symbol: "XRP",
+    amount: parseUnits("5427.9473", 18).toString(),
+    address: "0x1D2F0da169ceB9fC7B3144628dB156f3F6c60dBE",
+  },
+  {
+    symbol: "LINK",
+    amount: BigNumber.from(parseUnits("141.65625", 18)).add(parseUnits("75.03636", 18)).toString(),
+    address: "0xF8A0BF9cF54Bb92F17374d9e9A321E6a111a51bD",
+  },
+  {
+    symbol: "MATIC",
+    amount: parseUnits("2661.62045", 18).toString(),
+    address: "0xCC42724C6683B7E57334c4E856f4c9965ED682bD",
+  },
+  {
+    symbol: "BETH",
+    amount: parseUnits("0.71965", 18).toString(),
+    address: "0x250632378E573c6Be1AC2f97Fcdf00515d0Aa91B",
+  },
+  {
+    symbol: "DOGE",
+    amount: parseUnits("19727.91485", 8).toString(),
+    address: "0xbA2aE424d960c26247Dd6c32edC70B295c744C43",
+  },
+  {
+    symbol: "FIL",
+    amount: parseUnits("266.1287", 18).toString(),
+    address: "0x0D8Ce2A99Bb6e3B7Db580eD848240e4a0F9aE153",
+  },
+  {
+    symbol: "TUSD",
+    amount: BigNumber.from(parseUnits("1327.33975", 18)).add(parseUnits("3883.51888", 18)).toString(),
+    address: "0x40af3827F39D0EAcBF4A168f8D4ee67c121D11c9",
+  },
+  {
+    symbol: "wBETH",
+    amount: parseUnits("0.4473", 18).toString(),
+    address: "0xa2e3356610840701bdf5611a53974510ae27e2e1",
+  },
+  {
+    symbol: "DAI",
+    amount: parseUnits("4292.3044", 18).toString(),
+    address: "0x1AF3F329e8BE154074D8769D1FFa4eE058B1DBc3",
+  },
+  {
+    symbol: "TRX",
+    amount: parseUnits("11124.31032", 6).toString(),
+    address: "0xCE7de646e7208a4Ef112cb6ed5038FA6cC6b12e3",
+  },
+];
+
+const vBEP20TransferCommands = vToken_Transfers.map((vTokenTransfer: VTokenTransfer) => {
+  return {
+    target: TREASURY,
+    signature: "withdrawTreasuryBEP20(address,uint256,address)",
+    params: [vTokenTransfer.address, vTokenTransfer.amount, TOKEN_REDEEMER],
+  };
+});
+
+const vBEP20RedeemCommands = vToken_Transfers.map((vTokenTransfer: VTokenTransfer) => {
+  return {
+    target: TOKEN_REDEEMER,
+    signature: "redeemAndTransfer(address,address)",
+    params: [vTokenTransfer.address, TREASURY],
```

Comment:
This way treasury would have some leftover tokens. Maybe we could transfer them directly to Binance? If not, we could just redeem without TOKEN_REDEEMER contract 🤔